configurable_lti_consumer/configurable_lti_consumer.py

```diff
@@ -1,10 +1,18 @@
-from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 
 from lti_consumer import LtiConsumerXBlock
 
 from .exceptions import ConfigurableLTIConsumerException
 from .defaults import CONFIGURABLE_LTI_CONSUMER_SETTINGS
```

Comment:
this had to be removed for you solution to work right?

Reply:
Oops. Yeah, sorry for that.